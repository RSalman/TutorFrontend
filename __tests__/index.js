import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import TutoringApp from '../app/index';

it('renders correctly', () => {
  const tree = renderer.create(
    <TutoringApp />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
