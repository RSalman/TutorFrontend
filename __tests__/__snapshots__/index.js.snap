exports[`test renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      },
      undefined,
    ]
  }>
  <View
    style={
      Object {
        "flex": 1,
      }
    }>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      pointerEvents="auto"
      style={
        Object {
          "backgroundColor": "white",
          "bottom": 0,
          "left": 0,
          "opacity": 1,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 0,
            },
            Object {
              "translateY": 0,
            },
          ],
        }
      }>
      <View
        style={
          Array [
            Object {
              "flex": 1,
            },
            undefined,
          ]
        }>
        <Image
          source={1}
          style={
            Object {
              "flex": 1,
              "height": null,
              "width": null,
            }
          }>
          <View
            style={
              Object {
                "flex": 1,
                "justifyContent": "flex-end",
              }
            }>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "fontFamily": "HelveticaNeue-Thin",
                  },
                  Object {
                    "alignSelf": "center",
                    "backgroundColor": "transparent",
                    "color": "#008B8B",
                    "flex": 1,
                    "fontSize": 60,
                    "justifyContent": "flex-start",
                    "marginTop": 10,
                  },
                ]
              }>
              PROSPR
            </Text>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits="button"
              accessible={true}
              hitSlop={undefined}
              isTVSelectable={true}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "flexDirection": "row",
                  "height": 50,
                  "marginTop": 10,
                  "opacity": 1,
                  "overflow": "hidden",
                  "width": null,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}>
              <BVLinearGradient
                colors={
                  Array [
                    4283287460,
                    4284928890,
                  ]
                }
                end={
                  Object {
                    "x": 1,
                    "y": 1,
                  }
                }
                locations={null}
                start={
                  Object {
                    "x": 0.5,
                    "y": 1,
                  }
                }
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "flex": 1,
                    },
                    Object {
                      "borderRadius": 25,
                    },
                    undefined,
                  ]
                }>
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }>
                  <View
                    style={
                      Array [
                        Object {
                          "alignItems": "center",
                          "bottom": 0,
                          "flexDirection": "row",
                          "justifyContent": "center",
                          "left": 0,
                          "position": "absolute",
                          "right": 0,
                          "top": 0,
                        },
                        null,
                      ]
                    }>
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "white",
                          "flexDirection": "row",
                          "fontSize": 12,
                          "letterSpacing": 1,
                        }
                      }>
                      Tutors Component (Sarmad)
                    </Text>
                  </View>
                </View>
              </BVLinearGradient>
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits="button"
              accessible={true}
              hitSlop={undefined}
              isTVSelectable={true}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "flexDirection": "row",
                  "height": 50,
                  "marginTop": 10,
                  "opacity": 1,
                  "overflow": "hidden",
                  "width": null,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}>
              <BVLinearGradient
                colors={
                  Array [
                    4283287460,
                    4284928890,
                  ]
                }
                end={
                  Object {
                    "x": 1,
                    "y": 1,
                  }
                }
                locations={null}
                start={
                  Object {
                    "x": 0.5,
                    "y": 1,
                  }
                }
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "flex": 1,
                    },
                    Object {
                      "borderRadius": 25,
                    },
                    undefined,
                  ]
                }>
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }>
                  <View
                    style={
                      Array [
                        Object {
                          "alignItems": "center",
                          "bottom": 0,
                          "flexDirection": "row",
                          "justifyContent": "center",
                          "left": 0,
                          "position": "absolute",
                          "right": 0,
                          "top": 0,
                        },
                        null,
                      ]
                    }>
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "white",
                          "flexDirection": "row",
                          "fontSize": 12,
                          "letterSpacing": 1,
                        }
                      }>
                      Tutor Profile Component (Salman)
                    </Text>
                  </View>
                </View>
              </BVLinearGradient>
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits="button"
              accessible={true}
              hitSlop={undefined}
              isTVSelectable={true}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "flexDirection": "row",
                  "height": 50,
                  "marginTop": 10,
                  "opacity": 1,
                  "overflow": "hidden",
                  "width": null,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}>
              <BVLinearGradient
                colors={
                  Array [
                    4283287460,
                    4284928890,
                  ]
                }
                end={
                  Object {
                    "x": 1,
                    "y": 1,
                  }
                }
                locations={null}
                start={
                  Object {
                    "x": 0.5,
                    "y": 1,
                  }
                }
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "flex": 1,
                    },
                    Object {
                      "borderRadius": 25,
                    },
                    undefined,
                  ]
                }>
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }>
                  <View
                    style={
                      Array [
                        Object {
                          "alignItems": "center",
                          "bottom": 0,
                          "flexDirection": "row",
                          "justifyContent": "center",
                          "left": 0,
                          "position": "absolute",
                          "right": 0,
                          "top": 0,
                        },
                        null,
                      ]
                    }>
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "white",
                          "flexDirection": "row",
                          "fontSize": 12,
                          "letterSpacing": 1,
                        }
                      }>
                      Signup Form
                    </Text>
                  </View>
                </View>
              </BVLinearGradient>
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits="button"
              accessible={true}
              hitSlop={undefined}
              isTVSelectable={true}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "flexDirection": "row",
                  "height": 50,
                  "marginTop": 10,
                  "opacity": 1,
                  "overflow": "hidden",
                  "width": null,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}>
              <BVLinearGradient
                colors={
                  Array [
                    4283287460,
                    4284928890,
                  ]
                }
                end={
                  Object {
                    "x": 1,
                    "y": 1,
                  }
                }
                locations={null}
                start={
                  Object {
                    "x": 0.5,
                    "y": 1,
                  }
                }
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "flex": 1,
                    },
                    Object {
                      "borderRadius": 25,
                    },
                    undefined,
                  ]
                }>
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }>
                  <View
                    style={
                      Array [
                        Object {
                          "alignItems": "center",
                          "bottom": 0,
                          "flexDirection": "row",
                          "justifyContent": "center",
                          "left": 0,
                          "position": "absolute",
                          "right": 0,
                          "top": 0,
                        },
                        null,
                      ]
                    }>
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "white",
                          "flexDirection": "row",
                          "fontSize": 12,
                          "letterSpacing": 1,
                        }
                      }>
                      Leila\'s Feature
                    </Text>
                  </View>
                </View>
              </BVLinearGradient>
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits="button"
              accessible={true}
              hitSlop={undefined}
              isTVSelectable={true}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "flexDirection": "row",
                  "height": 50,
                  "marginTop": 10,
                  "opacity": 1,
                  "overflow": "hidden",
                  "width": null,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}>
              <BVLinearGradient
                colors={
                  Array [
                    4283287460,
                    4284928890,
                  ]
                }
                end={
                  Object {
                    "x": 1,
                    "y": 1,
                  }
                }
                locations={null}
                start={
                  Object {
                    "x": 0.5,
                    "y": 1,
                  }
                }
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "flex": 1,
                    },
                    Object {
                      "borderRadius": 25,
                    },
                    undefined,
                  ]
                }>
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }>
                  <View
                    style={
                      Array [
                        Object {
                          "alignItems": "center",
                          "bottom": 0,
                          "flexDirection": "row",
                          "justifyContent": "center",
                          "left": 0,
                          "position": "absolute",
                          "right": 0,
                          "top": 0,
                        },
                        null,
                      ]
                    }>
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "white",
                          "flexDirection": "row",
                          "fontSize": 12,
                          "letterSpacing": 1,
                        }
                      }>
                      Phone Verification (Sarmad)
                    </Text>
                  </View>
                </View>
              </BVLinearGradient>
            </View>
          </View>
        </Image>
      </View>
    </View>
  </View>
</View>
`;
